modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -579,14 +743,42 @@ inline IE::Blob::Ptr extractRemoteBlob(IECallContext& ctx, std::size_t i) {
 
 inline IE::Blob::Ptr extractBlob(IECallContext& ctx,
                                  std::size_t i,
-                                 cv::gapi::ie::TraitAs hint) {
+                                 cv::gapi::ie::TraitAs hint,
+                                 const std::string& layer_name,
```

Comment:
It's unnecessary to have `layer_name` here:

`const auto& layer_name = ctx->uu.params.input_names[i];`